.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        go: [ '1.16', '1.15', '1.14', '1.13' ]
+        go: [ '1.18', '1.17', '1.16' ]
```

Comment:
Could we go all the way to 1.19? I think it should be supported by actions/setup-go.

Reply:
Good thinking!  Can probably also bump to this in the go.mod file as well.